app/views/videos/show.html.haml

```diff
@@ -8,6 +8,9 @@
           %h3= @video.title 
           %span Rating: 4.5/5.0
         %p= @video.description
+        .actions
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id, user_id: current_user.id), 
```

Comment:
Why do you want to pass `current_user.id` to controller that is aware of the `current_user.id`?